modules/nw-ne-comparing-ingress-route.adoc

```diff
@@ -0,0 +1,11 @@
+// Module included in the following assemblies:
+//
+// * ingress/understanding-networking.adoc
+
+[id="nw-ne-comparing-ingress-route_{context}"]
+= Comparing Ingress and Routes
+The most common way to manage ingress traffic is with the Ingress Controller. OpenShift implements the Ingress Controller with a shared router service that runs as a pod inside the cluster. You can scale and replicate this pod like any other regular pod. This router service is based on the open source software HAProxy.
+
+Routes and ingress are the main resources for handling ingress traffic. An Ingress Controller is a Kubernetes resource that provides similar features to a route. Ingresses accept external requests and delegate them based on the route. You can only allow certain types of connections: HTTP/2, HTTPS and server name identification (SNI), and TLS with certificate. In OpenShift, routes are generated to meet the conditions specified by the ingress object.
+
+A route provides ingress traffic to services in the cluster. Routes were created before Kubernetes ingress objects and provide advanced features that may not be supported by standard Kubernetes ingress controllers, such as TLS re-encryption, TLS passthrough, and split traffic for blue-green deployments.
```

Comment:
Ingress traffic accesses service in the cluster through a route?

Reply:
yes, great clarifying detail thank you! I agree it does fit at the beginning.